scripts/e2e/setup_testnet.sh

```
@@ -211,210 +209,246 @@ generate_solver_configs() {
-
-[settlement.implementations.direct.routes]
-# Bidirectional routes - both chains can send to each other
-$origin_id = [$dest_id]
-$dest_id = [$origin_id]
-EOF
-
-    # Add configurations for all enabled settlement methods
-    append_enabled_settlements "$origin_id" "$dest_id"
+    local origin_rpc="$3"
+    local dest_rpc="$4"
+    local solver_id=$(get_config ".solver_parameters.id" "oif-solver-testnet")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)